lfric_styling.py

```diff
@@ -14,16 +14,19 @@
 
 Styling currently being applied:
     -lowercasing keywords
-
 '''
 
 import re
 import sys
 import argparse
 import os
+import threading
 from pathlib import Path
 from styling_keywords import NEW_KEYWORDS
```

Comment:
I think these can be removed as they don't seem to be called in the code. 

```suggestion
import re
import argparse
import os
from pathlib import Path
from styling_keywords import NEW_KEYWORDS
```